CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 ## Latest
 
+## 0.6.4
+
+### Important
+- Following components will be deprecated and archived in next release:
+  - clickhouse-cli-client
+  - clickhouse-grpc-client
+- No more builds for non-lts Java versions - no more Java 9 release builds.
+- Lowest supported Java version will be 11.
+  - Java 11 support will be ended before the end of 2023. 
+  - It is recommended to use Java 21. 
+
+### New Features 
+- [Client-V1/Apache HTTP] More configuration parameters for connection management. Useful for tuning performance.
+(https://github.com/ClickHouse/clickhouse-java/pull/1771)
+    - com.clickhouse.client.config.ClickHouseClientOption#CONNECTION_TTL - to configure connection time-to-live
+    - com.clickhouse.client.http.config.ClickHouseHttpOption#KEEP_ALIVE_TIMEOUT - to configure keep-alive timeout
+    - com.clickhouse.client.http.config.ClickHouseHttpOption#CONNECTION_REUSE_STRATEGY - to configure how connection 
```

Comment:
The setting name is self-descriptive. We'd rather focus on the meaning of the setting and possible configuration value. You can even use the existing comments from javadoc https://github.com/ClickHouse/clickhouse-java/blob/82f8b2125ab3f688bcfc9c44a2112394cf2aebb0/clickhouse-http-client/src/main/java/com/clickhouse/client/http/config/ClickHouseHttpOption.java#L93-L96

Reply:
Updated.